modules/op-constructing-pipelines-using-existing-tasks.adoc

```diff
@@ -0,0 +1,43 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
+
+[id="op-constructing-pipelines-using-existing-tasks_{context}"]
+= Constructing Pipelines using existing Tasks and ClusterTasks
+
+You can use the Pipeline Builder to:
+
+* Construct a Pipeline flow using existing Tasks. When you install the OpenShift Pipelines Operator, it adds reusable Pipeline ClusterTasks to your cluster.
+* Specify the type of Resources required for the Pipeline execution, and if required, add additional Parameters to the Pipeline.
+* Reference these Pipeline Resources in each of the Tasks in the Pipeline as Input and Output Resources.
+* The Parameters for a Task are prepopulated based on the specifications of the Task. If required, reference the additional Parameters added to the Pipeline in the Task.
+
+.Procedure
+
+. In the *Add* view of the *Developer* perspective, click the *Pipeline* tile to see the *Pipeline Builder* page.
+. Enter a unique name for the Pipeline.
+. Select a Task from the *Select task* list to add a Task to the Pipeline. For this example, select a *s2i-nodejs* Task.
+.. To add sequential Tasks to the Pipeline, click the + icon to the right or left of the Task, and from the *Select task* list, select the Task you want to add to the Pipeline. For this example, use the + icon to the right of the *s2i-nodejs* Task to add an *openshift-client* Task.
```

Comment:
Can we add a + icon image, instead of + sign?

Reply:
Simply replaced using the word plus instead of the symbol to try and use screenshots where it is likely to cause confusion.